package.xml

```diff
@@ -32,6 +32,8 @@
   <test_depend>launch</test_depend>
   <test_depend>ros2run</test_depend>
 
+  <group_depend>rosidlinterfaces_packages</group_depend>
```

Comment:
I think this needs to be `rosidl_interfaces_packages`

Reply:
hmmm Looks like I was unable to type :confounded: